src/test/java/com/metamx/metrics/SigarLoadTest.java

```diff
@@ -0,0 +1,18 @@
+package com.metamx.metrics;
+
+import com.metamx.emitter.service.ServiceEmitter;
+import java.lang.management.RuntimeMXBean;
+import junit.framework.Assert;
+import org.hyperic.sigar.Sigar;
+import org.hyperic.sigar.SigarException;
+import org.junit.Test;
+
+public class SigarLoadTest
+{
+  @Test
+  public void testSigarLoad() throws SigarException
+  {
+    Sigar sigar = SigarUtil.getSigar();
+    Assert.assertEquals(sigar.getCpuInfoList().length, Runtime.getRuntime().availableProcessors());
```

Comment:
Do these both draw from the same information? example: if I have a cpuset limited cgroup, will they both show the number of total cores, or will they only look at the number of cores available in the cpuset?